### PR TITLE
Updating PROVISIONER constants with new clinet cluster provisioner

### PR DESCRIPTION
**Pull request by suchita-g**
This will Fix issue 8889

### CODE REVIEW

**Comment by suchita-g** [comment]
verification Job:https://ocs4-jenkins-csb-odf-qe.apps.ocp-c1.prod.psi.redhat.com/job/qe-odf-multicluster/2112/

File: ocs_ci/ocs/constants.py [hunk 1]
```diff
@@ -314,7 +314,8 @@
 )
 DEFAULT_VOLUMESNAPSHOTCLASS_CEPHFS_MS = f"{DEFAULT_CLUSTERNAME}-cephfs"
 DEFAULT_VOLUMESNAPSHOTCLASS_RBD_MS = f"{DEFAULT_CLUSTERNAME}-ceph-rbd"
-
+DEFAULT_VOLUMESNAPSHOTCLASS_RBD_PC = f"{DEFAULT_CLUSTERNAME}-ceph-rbd"
```
Comment (fbalak) [inline ocs_ci/ocs/constants.py:317]: These constants are the same as DEFAULT_VOLUMESNAPSHOTCLASS_CEPHFS_MS and DEFAULT_VOLUMESNAPSHOTCLASS_RBD_MS, right?

**Review by jilju — APPROVED** [review]

File: ocs_ci/ocs/constants.py (hunk 1 above)
Comment (suchita-g) [reply ocs_ci/ocs/constants.py:317]: yes

Comment (fbalak) [reply ocs_ci/ocs/constants.py:317]: Would it make sense to reuse them? They seem to be the same volume snapshot classes.

Comment (suchita-g) [reply ocs_ci/ocs/constants.py:317]: may be to reuse it , we need to rename the existing one . doing it. Either a way can work. I see not used in multiple places so change is feasible.

**Comment by suchita-g** [comment]
Verification: https://ocs4-jenkins-csb-odf-qe.apps.ocp-c1.prod.psi.redhat.com/job/qe-odf-multicluster/2127/

**Review by fbalak — APPROVED** [review]

**Review by ebenahar — APPROVED** [review]

**Pull request merged**